packages/drivers/odsp-driver/src/test/odspDriverResolverTest.spec.ts

```diff
@@ -393,7 +390,7 @@ describe("Odsp Driver Resolver", () => {
 
 		assert.strictEqual(
 			resolvedUrl.url,
-			"fluid-odsp://placeholder/placeholder/AV5r7rhbMqs3T5cL8TUpqk6FpWldev0qKsKlnjkC5mg%3D/",
```

Comment:
Can you help me understand how this will manifest for a 1P partner using ODSP driver?  Where did this old protocol appear in practice and are there compat concerns?

Reply:
I put some analysis in the ADO item - it affects the value of `IResolvedUrl.url`, but likely not in a meaningful way for most scenarios.